components/catalogs/back/src/Application/Persistence/WarmupableQueryInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Catalogs\Application\Persistence;
+
+/**
+ * @copyright 2023 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+interface WarmupableQueryInterface
```

Comment:
```suggestion
interface WarmupAwareQueryInterface
```
maybe stupid :thinking:

Reply:
When I see `*Aware`, for me it's more that you have the `*` injected into it. WDYT?

Reply:
To me, it would mean the query is aware of the warm-up method (I'm not that familiar with `*Aware` convention)
It just grinds my (non) English ears when we slap `-able` to every word (**cough** `scopable` **cough**) :laughing: 
But feel free to ignore my comment :smile: